main.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/gorilla/mux"
+)
+
+//===========================================================================================
+//===========================================================================================
+//MODEL
+var db *sql.DB
+var err error
+
+/*create table users(Username varchar(10) primary,

```

Comment:
Create a separate .sql file and keep updating it with sql queries manually, or read about how to manage sql migration in golang.

Remove this comment from here.